docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -365,7 +365,7 @@ Release
         export VERSION=0.1.2
 
```

Comment:
Is it possible to use the latest-installable argument here as well, to allow removal of the export?